### PR TITLE
Adjust to new pulp3 pulp_url config

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: test/pulp3_api_test.rb [hunk 1]
```diff
@@ -14,7 +14,8 @@ def app
   end
 
   def test_returns_pulp_status_on_200
-    stub_request(:get, "#{::PulpProxy::Pulp3Plugin.settings.pulp_url.to_s}api/v3/status/").to_return(:body => "{\"api_version\":\"3\"}")
+    print ::PulpProxy::Pulp3Plugin.settings.pulp_url.to_s
```
Comment (ekohl) [inline test/pulp3_api_test.rb]: Looks like debug code

Comment (jlsherrill) [reply test/pulp3_api_test.rb]: doh!

Comment (jlsherrill) [reply test/pulp3_api_test.rb]: removed

**Comment by ekohl** [comment]
Thanks! Will you take care of a release?

**Comment by jlsherrill** [comment]
yes i'll handle release and packaging!

**Pull request merged**